src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -241,6 +241,23 @@ static bool kvsfs_fh_is_open(struct kvsfs_fsal_obj_handle *obj)
 	return (memcmp(&empty_share, &obj->share, sizeof(empty_share)) != 0);
 }
 
+/*****************************************************************************/
+/* Copy FSAL attrs into CORTXFS attrs
+ */
+inline void cortxfs_cred_from_op_ctx(cfs_cred_t *out)
+{
+	int i;
+
+	assert(out);
+
+	out->uid = op_ctx->creds->caller_uid;
+	out->gid = op_ctx->creds->caller_gid;
+	out->total_grps = op_ctx->creds->caller_glen;
+	for (i=0; i<op_ctx->creds->caller_glen; i++) {
```

Comment:
In current code, where we are getting the group list from NFS client via RPC, 16 is the upper limit. So no need to check.
In case of "manage_gids" option, there can be more groups. This is not supported as of now.